lib/util/parseFile.ts

```diff
@@ -83,27 +83,47 @@ const getChange = (
     };
   }
 
-  // we want to correctly remove 'default' when its a default export so we get the syntaxList node instead of the exportKeyword node
-  // note: the first syntaxList node should contain the export keyword
+  /**
+   * The syntax list contains the keywords that can be found before the actual declaration.
+   * We want to remove everything that is not a decorator.
+   */
   const syntaxListIndex = node
     .getChildren()
     .findIndex((n) => n.kind === ts.SyntaxKind.SyntaxList);
 
-  const syntaxList = node.getChildren()[syntaxListIndex];
+  const syntaxList = node?.getChildren()[syntaxListIndex];
+
+  if (!syntaxList) {
+    throw new Error('Syntaxlist missing');
```

Comment:
```suggestion
    throw new Error('sytaxList missing');
```

or maybe `syntax list` [nit]

Reply:
done